pkg/http/api.go

```diff
@@ -130,6 +130,12 @@ func (a *api) constructStateEndpoints() []Endpoint {
 			Version: apiVersionV1,
 			Handler: a.onDeleteState,
 		},
+		{
+			Methods: []string{fasthttp.MethodPost},
+			Route:   "state/{storeName}/bulk",
```

Comment:
Will it route correctly if key happens to be "bulk" ? Could that possibly route to onGetState ?

Reply:
It does not collide with onGetState because the verb for bulk is POST while the other is GET.

Reply:
Can we use POST verb for Get semantic ? - as I think POST is for set semantic..

Reply:
We cannot use GET for this.